.github/workflows/prepare-release.yml

```diff
@@ -42,7 +56,7 @@ jobs:
 
       - name: Commit release preparation changes
         run: |
-          git -c user.name='github-actions[bot]' -c user.email='github-actions[bot]@users.noreply.github.com' \
+          git -c user.name='cvat-bot[bot]' -c user.email='147643061+cvat-bot[bot]@users.noreply.github.com' \
```

Comment:
What that number is?

Reply:
It's the user ID of the app. You can see it here.

https://api.github.com/users/cvat-bot[bot]